src/aave-v3/SupplyVault.sol

```diff
@@ -176,16 +166,20 @@ contract SupplyVault is ISupplyVault, SupplyVaultBase {
             address(this),
             _rewardToken
         );
+
         UserRewardsData memory rewards = userRewards[_rewardToken][_user];
+        uint256 supply = totalSupply();
```

Comment:
better IMO
```suggestion
        uint256 totalSupply = totalSupply();
```

Reply:
`totSupply` perhaps? Else we have a variables shadowing that I don't really like

Reply:
`totalSupplyStack` ? `totalSupplyCache` ?

Reply:
easy to confuse with the user's supply, but ok